core/gather/driver/execution-context.js

```diff
@@ -16,6 +16,12 @@ class ExecutionContext {
 
     /** @type {number|undefined} */
     this._executionContextId = undefined;
+    /**
+     * Marks the order that execution context ids are used, for purposes of having a unique
+     * value (that doesn't expose the actual execution context id) to use for __lighthouseMainExecutionContextId.
+     * @type {number[]}
+     */
+    this._executionContextIdentifiersUsed = [];
```

Comment:
Initially I just did `contextId - this._executionContextId`, but this is problematic due to the existence of `clearContextId`.